server/views/pages/about.njk

```diff
@@ -75,7 +75,7 @@
                 </tr>
 
                 {% for sentence in data.deliusData.sentences %}
-                    <tr class="govuk-table__row govuk-body-l about-table-body">
+                    <tr class="govuk-table__row govuk-body-l {% if data.deliusData.sentences.length == 1 %}single-sentence{% endif %}">
```

Comment:
```suggestion
                    <tr class="govuk-table__row govuk-body-l {{ 'single-sentence' if data.deliusData.sentences.length == 1 }}">
```

I think you can shorten it to this (though I would give it a quick test!)

Reply:
yes that looks plausible, I'll check it out.